unstructured_inference/inference/layout.py

```diff
@@ -96,7 +140,7 @@ def __init__(
         self,
         number: int,
         image: Image,
-        layout: Optional[lp.Layout],
+        layout: Optional[lp.Layout],  # Add: List[LayoutElement]],
```

Comment:
What's the `# Add: List[LayoutElement]],` comment?

Reply:
Oh, my mistake: but was referring to the .elements a few lines below